code/modules/mob/living/carbon/alien/alien_base.dm

```diff
@@ -220,14 +220,18 @@ Des: Removes all infected images from the alien.
 	return
 
 /mob/living/carbon/alien/canBeHandcuffed()
-	return 1
+	return TRUE
 
-/mob/living/carbon/alien/proc/updatePlasmaDisplay()
-	if(hud_used) //clientless aliens
-		hud_used.alien_plasma_display.maptext = "<div align='center' valign='middle' style='position:relative; top:0px; left:6px'> <font face='Small Fonts' color='magenta'>[getPlasma()]</font></div>"
-		hud_used.alien_plasma_display.maptext_x = -3
+/* Although this is on the carbon level, we only want this proc'ing for aliens that do have this hud. Only humanoid aliens do at the moment, so we have a check
+and carry the owner just to make sure*/
+/mob/living/carbon/proc/updateplasmadisplay(mob/owner)
+	RegisterSignal(owner, COMSIG_MIND_TRANSER_TO)
```

Comment:
Why have a register here without a proc to call?

Reply:
I don't remember exactly what, but I did this to make something work
I'm pretty sure it was for updating the plasma display

Reply:
Nevermind not needed I handled this somewhere else